api/tests/authentication_tests.py

```diff
@@ -213,13 +224,63 @@ def test_get_fxa_uid_from_oauth_token_not_active_uses_cached_response_returns_er
             return
         self.fail("Should have raised AuthenticationFailed")
 
+    @responses.activate
+    def test_get_fxa_uid_from_oauth_token_has_wrong_scope_returns_error_response(  # noqa: E501
+        self,
+    ) -> None:
+        now_time = int(datetime.now().timestamp())
+        # Note: FXA iat and exp are timestamps in *milliseconds*
+        exp_time = (now_time + 60 * 60) * 1000
+        json_data: FxaResponse = {
+            "active": True,
+            "sub": self.uid,
+            "exp": exp_time,
+            "scope": "foo",
+        }
+        status_code = 200
+        fxa_response = _setup_fxa_response(status_code, json_data)
+        missing_scopes_token = "missing-scopes-123"
+        cache_key = get_cache_key(missing_scopes_token)
+
+        assert cache.get(cache_key) is None
+
+        # get fxa response with no fxa uid for the first time
+        try:
+            get_fxa_uid_from_oauth_token(missing_scopes_token)
+        except AuthenticationFailed as e:
+            assert (
+                str(e.detail)
+                == "FXA missing is missing scope: https://identity.mozilla.com/apps/relay."
```

Comment:
suggestion (non-blocking):

```suggestion
                == "FXA token is missing scope: https://identity.mozilla.com/apps/relay."
```